Enable unicode normalization by default for speech (#17017)
nvaccess/nvda: user_docs/en/changes.md

* The `-c`/`--config-path` and `--disable-addons` command line options are now respected when launching an update from within NVDA. (#16937)
* eSpeak NG has been updated to 1.52-dev commit `961454ff`. (#16775)
  * Added new languages Faroese and Xextan.

> * The default state of Unicode Normalization for speech output has been changed to enabled. (#17017, @LeonarddeR).

Reviewer: ```suggestion
* Unicode Normalization is now enabled by default for speech output. (#17017, @LeonarddeR).
```

>   * You can still disable this functionality in the speech settings categorie in the NVDA Settings dialog.

Reviewer: ```suggestion
  * You can still disable this functionality in the speech settings category in the NVDA Settings dialog.
```

Reviewer 2: ```suggestion
  * You can still disable this functionality in the Speech category of the NVDA Settings dialog.
```

* The `-c`/`--config-path` and `--disable-addons` command line options are now respected when launching an update from within NVDA. (#16937)
* eSpeak NG has been updated to 1.52-dev commit `961454ff`. (#16775)
  * Added new languages Faroese and Xextan.
* Unicode Normalization is now enabled by default for speech output. (#17017, @LeonarddeR).

>   * You can still disable this functionality in the Speech category of the NVDA Settings dialog.

Reviewer: this needs to be moved to 2025.1

* eSpeak NG has been updated to 1.52-dev commit `961454ff`. (#16775)
  * Added new languages Faroese and Xextan.
* When using a multi-line braille display via the standard HID braille driver, all lines of cells will be used. (#16993, @alexmoon)

> * The stability of NVDA's Poedit support has been improved with the side effect that the minimum required version of Poedit is now version 3.5. (#16889, @LeonarddeR)

Reviewer: please remove the change to 2024.4 change log

Reviewer 2: No idea how that got deleted, whoops.
